modules/userId.js

```diff
@@ -175,13 +206,22 @@ export function getStoredValue(storage) {
   return storedValue;
 }
 
+/**
+ * test if consent module is present, and if GDPR applies
+ * @param {Object} consentData
+ * @returns {boolean}
+ */
+export function GDPRApplies(consentData) {
```

Comment:
Find a function name that allows the first character to be lower-case. Something like: 
`isGDPRApplicable` or `doesGDPRApply`

Reply:
Right, renamed to isGDPRApplicable